api/app/files/models.py

```diff
@@ -0,0 +1,9 @@
+from app import db
+
+class File(db.Model):
+	__tablename__ = "file"
+	id = db.Column(db.Integer, primary_key=True)
+	file_name = db.Column(db.String(50), unique=True, nullable=False)
+	mime_type = db.Column(db.String(20), nullable=False)
+	guid = db.Column(db.String(50), unique=True, nullable=False)
```

Comment:
I think this model is fine. Perhaps add a constructor though. It just results in cleaner code when want to add a new row in the database and allows us to have a consistent way of creating a file object.

```
def __init__(self, file_name, mime_type):
  self.file_name = file_name
  self.mime_type = mime_type
  self.guid = str(uuid.uuid4().hex)
```

Then have a look at the [Database Migrations](https://github.com/deep-learning-indaba/Baobab/blob/develop/api/README.md) section of the docs. Basically these classes (that inherit from `db.Model`) represent a table in the database so we actually need to make sure a table exists in the database. The migration will create the script that will actually go and create this table then. Let me know if the docs aren't clear. They're a work in progress so it definitely helps to know if something doesn't make sense.